src/Twig/AppExtension.php

```diff
@@ -88,4 +91,21 @@ public function getLocales(): array
 
         return $this->locales;
     }
+
+    public function getPaginator(Paginator $paginator, int $currentPage, int $pageSize = Post::NUM_ITEMS): array
+    {
+        $hasPreviousPage = $currentPage > 1;
+        $hasNextPage = ($currentPage * $pageSize) < $paginator->count();
+
+        return [
+            'results' => $paginator,
```

Comment:
should do `$paginator->getIterator()` to have the right count of elements in the results.